crates/sqlbuiltins/src/functions/mod.rs

```diff
@@ -337,8 +396,50 @@ impl FunctionRegistry {
         }
         None
     }
+
+    fn register_udf(&mut self, f: Arc<dyn BuiltinScalarUDF>) {
+        let keys = match f.namespace() {
+            // we register the function under both the namespaced entry and the normal entry
+            // e.g. select foo.my_function() or select my_function()
```

Comment:
does it make sense to only register the legacy/existing functions in the global namespace, and put everything else in a namespace?